### PR TITLE
Player styling

### PR DESCRIPTION
**Pull request by gcampo88**
Addresses #48. 
- [x] added styles to player component to match designs for single episode
- [x] added svg files for various control buttons
- [x] added a few html elements / reordered others to allow for blurred background image and reordered controls

### CODE REVIEW

**Comment by kookster** [comment]
@farski do you have time to review?

File: src/images/large-semi-transparent-pause.svg [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="60px" height="60px" viewBox="0 0 60 60" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: sketchtool 40.3 (33839) - http://www.bohemiancoding.com/sketch -->
```
Comment (farski) [inline src/images/large-semi-transparent-pause.svg]: Line 3, 4, 5, and 6 in all these SVG's can be removed

File: src/app/+player/player.component.html [hunk 2]
```diff
@@ -1,39 +1,59 @@
...
-      [value]="currentTime | async"
-      [maximum]="duration | async"
-      (seek)="onSeek($event)"
-      (scrubbing)="onScrub($event)"></progress-bar>
-    <playback-progress>{{currentTime | async | duration}}</playback-progress>
-    <duration>{{duration | async | duration}}</duration>
-  </scrubber>
-</controls>
+
+<background-img [style.background-image]="artworkUrl"></background-img>
```
Comment (gcampo88) [inline src/app/+player/player.component.html]: as discussed!: ng did not like: `Template parse errors:
Only void and foreign elements can be self closed "background-img" ("[ERROR ->]<background-img [style.background-image]="artworkUrl" />`

File: src/app/+player/player.component.html [hunk 3]
```diff
@@ -1,39 +1,59 @@
...
+  <content>
```
Comment (gcampo88) [inline src/app/+player/player.component.html]: 👍

File: src/images/large-semi-transparent-pause.svg (hunk 1 above)
Comment (gcampo88) [inline src/images/large-semi-transparent-pause.svg]: 👍

**Comment by gcampo88** [comment]
@farski: addressed all, I believe!

**Pull request merged**